applications/openshift/controller/controller_root-ca-file/rule.yml

```diff
@@ -0,0 +1,40 @@
+documentation_complete: true
+
+title: 'Ensure that the --root-ca-file argument is set'
+
+description: |-
+    To ensure pods verify the API Server's serving certificate before
+    establishing connections, edit the Controller Manager pod specification
+    file <tt>/etc/kubernetes/manifests/kube-controller-manager.yaml</tt>
+    on the master node(s) and set the <tt>--root-ca-file</tt> parameter
+    to the certificate bundle file. For example:
+
+    <pre>--root-ca-file=</path/to/file></pre>
```

Comment:
It doesn't build because it is confused by the `<` and `>` characters here, it thinks `</path/to/file>` is as an HTML tag.